static/js/containers/PaymentPage.js

```diff
@@ -247,6 +247,7 @@ class PaymentPage extends React.Component {
             payment={payment}
             payableKlassesData={payableKlassesData}
             selectedKlass={selectedKlass}
+            now={moment()}
```

Comment:
We use a similar pattern of calculating `now` in the `render` method in MM too (ex: `CourseListCard`). This value is calculated when the component is rendered, and the changing time will not cause a state change that would cause a re-render